server/config/type.js

```diff
@@ -221,6 +236,25 @@ const ConfigType = class ConfigType {
         formattedConfig[relationName] = relations;
       }));
 
+      this.populate
+        .filter((populatedFields) => !populatedFields.includes("."))
+        .map((populatedFields) => {
+            formattedConfig[populatedFields] = formattedConfig[
+              populatedFields
+            ].map((fields) => {
+              const sanitizeObjects = (fields) => {
+                sanitizeConfig(fields);
+                Object.keys(fields).map((key, index) => {
+                  if (fields[key] && typeof fields[key] === "object") {
+                    sanitizeObjects(fields[key]);
```

Comment:
This recursive config sanitization is great!

Could you add this directly to the `sanitizeConfig` utility.
That way we benefit from this logic on all places where we're using the `sanitizeConfig` utility.

Reply:
I have moved the logic according to your request ;)

Reply:
Thanks. That's great!
There is just one eslint issue with this implementation.

Please see the pipeline output:
https://github.com/boazpoolman/strapi-plugin-config-sync/actions/runs/5819068278/job/15815463391?pr=93